go/roothash/api/api.go

```diff
@@ -571,6 +571,10 @@ type ConsensusParameters struct {
 
 	// MaxEvidenceAge is the maximum age of submitted evidence in the number of rounds.
 	MaxEvidenceAge uint64 `json:"max_evidence_age"`
+
+	// MaxPastRootsStored is the maximum number of past runtime state and I/O
+	// roots that are stored in the consensus state.
+	MaxPastRootsStored uint32 `json:"max_past_roots_stored,omitempty"`
```

Comment:
Also update fixgenesis (and the test runner's default genesis generation) to configure a sensible value for this (e.g. it should default to something like 1200 rounds).